File: 0014-longest-common-prefix/0014-longest-common-prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        int n=strs.size();
        if(n==0)
            return "";
        string ans="";
        sort(begin(strs),end(strs));
        string f=strs[0];
        string l=strs[n-1];
        for(int i=0;i<f.size();i++){
            if(f[i]==l[i]){
                ans+=f[i];
            }
            else{
                break;
            }
        }
        return ans;
    }
};

File: 0014-longest-common-prefix/0014-longest-common-prefix.java
import java.util.Arrays;
class Solution {
    public String longestCommonPrefix(String[] strs) {
        StringBuilder sb=new StringBuilder();
        Arrays.sort(strs);
        int n=strs.length;
        char[] first=strs[0].toCharArray();
        char[] last=strs[n-1].toCharArray();

        for(int s=0;s<first.length;s++){
            if(first[s]!=last[s]){
                break;
            }
            sb.append(first[s]);
        }
        return sb.toString();
    }
}